core/server/apps/amp/lib/router.js

```diff
@@ -42,21 +43,42 @@ function getPostData(req, res, next) {
             next();
         })
         .catch(function (err) {
-            if (err instanceof errors.NotFoundError) {
-                return next(err);
-            }
-
             next(err);
         });
 }
 
+function checkIfAMPIsEnabled(req, res, next) {
+    var ampIsEnabled = config.theme.amp;
+
+    if (ampIsEnabled) {
+        return next();
+    }
+
+    // CASE: we don't support amp pages for static pages
+    if (req.body.post && req.body.post.page) {
+        return errors.error404(req, res, next);
+    }
+
+    /**
+     * CASE: amp is disabled, we redirect the user to the requested post
```

Comment:
Yeah, know about the link (I built it in 😉), wasn't sure if that's enough! But good to know! Thanks a lot, @kevinansfield! 🤗

And yes, totally forgot to remove this link... Sorry!